async-nats/src/jetstream/stream.rs

```diff
@@ -77,6 +77,46 @@ impl Stream {
         }
     }
 
+    /// Retrieves `info` about [Stream] from the server, updates the cached `info` inside
```

Comment:
Maybe it would be good to add info about `deleted` not being populated with IDs of deleted messages with lowest sequence numbers?